Submissions/@kabba1.json

```diff
@@ -0,0 +1,16 @@
+{
+    "name": "Mahamadou Kabba",
+    "img": "https://media.licdn.com/dms/image/D4E03AQHucjMrnafYpw/profile-displayphoto-shrink_200_200/0/1681159796384?e=1686787200&v=beta&t=Lf5lRUBUlmss0fiakbJUGIIhNC-wYRS4JqAoMBIhWNA",
+    "email": "mkabba34@gmail.com",
+    "links": {
+      "website": "mkabba.com",
```

Comment:
please make this the full URL

Reply:
Hello, I'm now seeing this. what exactly do you need me to do here? make what the url?

Reply:
nvm, I fixed it